### DISCUSSION TITLE
SceneType' requires that 'ViewController' inherit from 'UIViewController'

### DISCUSSION THREAD

**Discussion by ARIEnergy** [SwiftGen/SwiftGen#973] [General]
Actually my ViewController inherit from UIViewController, and then I run swiftgen ,but it always showed the error

**Comment by ARIEnergy** [comment]
@djbe  Please help me to fix the terrible bug....

@AliSoftware  Any tips to fix the bug??? Thank you so much!

**Reply by mickyzinho** [reply]
Did you find a solution? Having the same issue.

**Comment by djbe** [comment]
You'll need to provide more information, like what your configuration looks like, and maybe your storyboard file. Do you have a small test case that reproduces the error?

**Comment by kienvanba** [comment]
In case anyone running into this issue in the future, I have the same issue and it was because of my **Storyboard** name is the same with my **ViewController** name.
In my case I have a storyboard name `FooViewController.storyboard` and inside that storyboard I have a screen named `FooViewController`. Changing the story board name to `Foo.storyboard` solved the problem for me
